platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -340,18 +381,18 @@ public extension WysiwygComposerViewModel {
         case let .update(newState):
             if newState[.link] != actionStates[.link] {
                 applyUpdate(update, skipTextViewUpdate: true)
-                textView.apply(attributedContent)
+                applyAtributedContent()
                 updateCompressedHeightIfNeeded()
                 return false
             }
         default: break
         }
         
-        applyUpdate(update, skipTextViewUpdate: shouldAcceptChange)
-
-        return shouldAcceptChange
+        applyUpdate(update, skipTextViewUpdate: skipTextViewUpdate)
+        lastReplaceTextUpdate = nextTextUpdate
+        return skipTextViewUpdate
```

Comment:
From the outside, this change in return is hard to understand given that the nature of this property is inverted, but there's no doc comments to explain what it is meant to signal to the caller.

Reply:
`shouldAcceptChange ` seemed a bit unclear to me given the language on line 391 was 

```
applyUpdate(update, skipTextViewUpdate: shouldAcceptChange)
```

So I was trying to make it consistent.

I've documented the return value in the function signature. Does that clear things up?

Reply:
Yep yep 👍